tests/conftest.py

```diff
@@ -164,6 +164,14 @@ def tear_down_hermitian():
     qml.Hermitian._eigs = {}
 
 
+@pytest.fixture
+def in_tape_mode():
+    """Run the test in tape mode"""
+    qml.enable_tape()
+    yield
+    qml.disable_tape()
+
+
 @pytest.fixture(params=[False, True])
 def tape_mode(request, mocker):
     """Tests using this fixture will be run twice, once in tape mode and once without."""
```

Comment:
The problem is, we use `only_tape_mode` for a fixture elsewhere, that _skips_ a test if tape mode is off 🤔 

Perhaps best not to worry too much about the naming too much for now, since these fixtures are (hopefully!) very temporary.